### ISSUE TITLE
Update nibabel pin

### ISSUE THREAD

**Issue by oesteban** [poldracklab/niworkflows#458]
We are currently getting:

```
ERROR: niworkflows 1.1.5 has requirement nibabel~=3.0.0rc1, but you'll have nibabel 3.0.1 which is incompatible.
```

**Comment by oesteban** [comment]
Fixed with 3af974a390486da6f9618249ba4094450d4c8491

**Comment by mgxd** [comment]
we'll also have to upstream this to fmriprep
